lom/src/lib/lomcommon/yang_parser/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+    "fmt"
+    "lom/src/lib/lomcommon/yang_parser/yang_utils"
+    "os"
+)
+
+const (
+    device_health_actions_configs           string = "device-health-actions-configs"
+    device_health_actions_configs_file_name string = "device-health-actions-configs.yang"
+
+    device_health_bindings_configs           string = "device-health-bindings-configs"
+    device_health_bindings_configs_file_name string = "device-health-bindings-configs.yang"
+
+    device_health_globals_configs           string = "device-health-global-configs"
+    device_health_globals_configs_file_name string = "device-health-global-configs.yang"
+
+    device_health_procs_configs           string = "device-health-procs-configs"
+    device_health_procs_configs_file_name string = "device-health-procs-configs.yang"
+
+    actions_conf_file_name  string = "actions.conf.json"
```

Comment:
Can we now it self change actions.conf.d/* ?
We are not blocked. 
We have time. As well get it right in first cut.